### PR TITLE
Update for containerized purge cluster playbook

### PR DESCRIPTION
**Pull request by font**
- Added support for purging containerized rbd-mirror node

Signed-off-by: Ivan Font <ivan.font@redhat.com>

### CODE REVIEW

File: purge-docker-cluster.yml [hunk 1]
```diff
@@ -130,6 +130,49 @@
...
+
+  hosts:
+    - "{{ rbdmirror_group_name }}"
+
+  become: true
+
+  tasks:
+  - include_vars: roles/ceph-common/defaults/main.yml
+  - include_vars: roles/ceph-rbd-mirror/defaults/main.yml
+  - include_vars: group_vars/all
```
Comment (leseb) [inline purge-docker-cluster.yml]: `- include_vars: group_vars/rbd-mirrors`

File: purge-docker-cluster.yml [hunk 2]
```diff
@@ -130,6 +130,49 @@
...
+  - name: remove ceph rbd-mirror container
+    docker:
+      image: "{{ ceph_rbd_mirror_docker_username }}/{{ ceph_rbd_mirror_docker_imagename }}:{{ ceph_rbd_mirror_docker_image_tag }}"
+      name: "{{ ansible_hostname }}"
+      state: absent
+    ignore_errors: true
+
+  - name: remove ceph rbd-mirror service
+    file:
+        path: /etc/systemd/system/ceph-rbd-mirror@.service
```
Comment (leseb) [inline purge-docker-cluster.yml]: indent of 2

**Comment by font** [comment]
@leseb ping

**Comment by leseb** [comment]
@font could you please rebase your change? thanks!

**Comment by font** [comment]
@leseb No problem - updated with rebase.

**Pull request merged**